test/src/e2e_vm_tests/test_programs/should_pass/language/raw_ptr/raw_ptr_ret/test.toml

```diff
@@ -1,2 +1,2 @@
 category = "run"
-expected_result = { action = "return", value = 10648 }
+expected_result = { action = "return", value = 10680 }
```

Comment:
@ironcev : Is this a meaningful test when the result can change with version updates?

Reply:
this is changed while bumping to 0.23.0 as well -- https://github.com/FuelLabs/sway/pull/5759/files#diff-b3e55e3e9349a52e1e1a886997e30212e5bcb273631949e98f469f195c38566a

Reply:
This is normal, returning raw pointer values depends on how the virtual machine allocates memory and isn't deterministic across versions.

This is why we don't support returning them through the ABI in the new encoding.

This test will probably break when we stabilize that, but we should still hack it back to working because figuring out when there are significant memory allocation changes is still valuable.

In any case, we can merge this update, it's valid I think.